litellm/utils.py

```diff
@@ -5849,6 +5849,7 @@ def _map_and_modify_arg(supported_params: dict, provider: str, model: str):
         for k in passed_params.keys():
             if k not in default_params.keys():
                 optional_params[k] = passed_params[k]
+    if custom_llm_provider == "openai" and "qwen" in model: optional_params.pop("user", None)
```

Comment:
@allen12921 we have a list of 'openai_compatible_providers', can you add it there instead - https://github.com/BerriAI/litellm/blob/514c5737f8f850d8309291107b1bb760c787c482/litellm/__init__.py#L379

And add a working example to docs - e.g. - https://github.com/BerriAI/litellm/blob/514c5737f8f850d8309291107b1bb760c787c482/docs/my-website/docs/providers/groq.md#L4